google/cloud/pubsub/internal/tracing_pull_lease_manager_test.cc

```diff
@@ -83,6 +85,39 @@ TEST(TracingPullLeaseManagerImplTest, AsyncModifyAckDeadlineSuccess) {
                          SpanNamed("test-subscription modack"))));
 }
 
+#if OPENTELEMETRY_VERSION_MAJOR >= 2 || \
+    (OPENTELEMETRY_VERSION_MAJOR == 1 && OPENTELEMETRY_VERSION_MINOR >= 13)
+TEST(TracingPullLeaseManagerImplTest, CreateRootSpan) {
+  auto span_catcher = InstallSpanCatcher();
+  auto active_span = MakeSpan("active span");
+  auto active_scope = opentelemetry::trace::Scope(active_span);
+  active_span->End();
```

Comment:
I imagine this should be done after the modack call

Reply:
Yup, fixed.